SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Microsoft.Azure.Mobile.Windows.Shared.projitems

```diff
@@ -20,6 +20,8 @@
     <Compile Include="$(MSBuildThisFileDirectory)Channel\IChannelUnit.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Ingestion\Http\HttpIngestionException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Ingestion\Http\IHttpNetworkAdapter.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Storage\IStorageAdapter.cs" />
```

Comment:
At the time of my last comment, the only change to this file is moving 'IStorageAdapter.cs' and 'StorageAdapter.cs' around, not an actual change. That's why I said you can undo this file. Now I saw you actual have some change, so you can ignore me. 😄